genrl/deep/agents/dqn/base.py

```diff
@@ -0,0 +1,246 @@
+from copy import deepcopy
+from typing import Any, Dict, List
+
+import numpy as np
+import torch
+from torch import optim as opt
+from torch.nn import functional as F
+
+from genrl.deep.agents.base import OffPolicyAgent
+from genrl.deep.common.utils import get_env_properties, get_model, safe_mean
+
+
+class BaseDQN(OffPolicyAgent):
```

Comment:
Why is this `BaseDQN`? Why not just `DQN`?

`BaseDQN` sounds a bit off, sounds like some ABC. I think you can change it to just `DQN` because all other are **extra** additions to the normal `DQN`

Reply:
Yeah, changed this to `DQN`. You were right, the extra class is redundant.